internal/a3m/a3m.go

```
@@ -81,28 +82,49 @@ func (a *CreateAIPActivity) Execute(ctx context.Context, opts *CreateAIPActivity
 				if err != nil {
 					return err
 				}
-
+				if reflect.ValueOf(a.cfg.Name).IsZero() {
+					a.cfg.Name = "enduro"
+				}
+				if reflect.ValueOf(a.cfg.Processing).IsZero() {
+					a.cfg.Processing = Processing{
+						AssignUuidsToDirectories:                     true,
+						ExamineContents:                              false,
+						GenerateTransferStructureReport:              true,
+						DocumentEmptyDirectories:                     true,
+						ExtractPackages:                              true,
+						DeletePackagesAfterExtraction:                false,
+						IdentifyTransfer:                             true,
+						IdentifySubmissionAndMetadata:                true,
+						IdentifyBeforeNormalization:                  true,
+						Normalize:                                    true,
+						TranscribeFiles:                              true,
+						PerformPolicyChecksOnOriginals:               true,
+						PerformPolicyChecksOnPreservationDerivatives: true,
+						AipCompressionLevel:                          1,
+						AipCompressionAlgorithm:                      a3m_transferservice.ProcessingConfig_AIP_COMPRESSION_ALGORITHM_S7_BZIP2,
+					}
+				}
```

To be clear, the zero value of a type such Processing is an instance where all its fields are set to their respective zero values, e.g. `Normalize: false`, `AipCompressionLevel: 0`, etc... but if someone purposely sets all fields to zero this block would overwrite the user's config with the defaults you're proposing, that would be misleading.

If you want to provide configuration defaults I think you'd need to [establish defaults using Viper's API](https://pkg.go.dev/github.com/spf13/viper#readme-establishing-defaults). You could try to declare a new `ProcessingDefaults` in the `a3m` package and refer to it from `internal/config/config.go` (`Read`), e.g.:

```go
v.SetDefault("a3m.processing", a3m.ProcessingDefaults)
```